chapters/en/slides/module5/module5_03.Rmd

```diff
@@ -0,0 +1,316 @@
+---
+params:
+  dynamictitle: "module5_03"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+
+# Defining and transforming axis ranges
+
+Notes:
+In this slide deck,
+we will see how defining an explicit axis range
+is beneficial when we need to zoom into a region of a plot,
+and how log transforming an axis can help increase resolution of small values.
+
+---
+
+## Read in the data containing the daily sum of donations to Wikipedia in 2020
+
+```{python}
+# TODO save wrangled data?
+import pandas as pd
+import altair as alt
+from vega_datasets import data
+
+donations = (
+    pd.read_csv("https://frdata.wikimedia.org/donationdata-vs-day.csv", parse_dates=['date'])
+    .query('2020 < date < 2021')
+    .assign(week_day=lambda df: df['date'].dt.day_name().str[:3]))
+
+alt.Chart(donations, title='Wikipedia donations peak during Christmas').mark_circle().encode(
+    alt.X('date', title=None),
+    alt.Y('sum', axis=alt.Axis(format='$s'), title='Amount donated per day'))
+```
+
+Notes:
+We will be working with a data set that contains the daily sum
```

Comment:
Can we start this transcript by posing a question we might be interested in answering here?